DEPENDENCIES

```diff
@@ -204,6 +204,88 @@ Prerequisites for building the OpenDataPlane (ODP) API
    1024MB of memory:
    $ sudo ODP_PKTIO_DPDK_PARAMS="-m 1024" ./test/performance/odp_l2fwd -i 0 -c 1
 
+3.5 Code instrumentation with PAPI library (optional)
+
+   PAPI library is used to instrument ODP API.
+
+3.5.1 Building PAPI library
+
+   ODP example library 'instrum' is using PAPI library to retrieve performance
+   counters associated with execution of ODP API. PAPI library is currently at 5.6.0.
+   However, if possible one should try to use the latest PAPI master branch commit for
+   the best performance and the latest bug fixes.
```

Comment:
Is the difference between 5.6 and master _that_ huge to warrant special notice?

Reply:
This part is more or less copy/paste from '3.3.1 Building netmap kernel modules'. Yet, it is important to mention the PAPI library version on which code instrumentation runs successfully and the possibility to use newer versions if available and backward compatible.   
PAPI 5.6.0 was released on 2017-12-20 (see http://icl.cs.utk.edu/papi/)

Reply:
- Ubuntu 16.04 LTS has PAPI 5.4, bionic has PAPI 5.6
- Debian stable has PAPI 5.5, testing has PAPI 5.6

I still think that we should suggest using distro-provided PAPI, with fallback to compiling it from source if required.